packages/flutter/lib/src/widgets/interactive_viewer.dart

```diff
@@ -11,8 +11,17 @@ import 'package:vector_math/vector_math_64.dart' show Quad, Vector3, Matrix4;
 import 'basic.dart';
 import 'framework.dart';
 import 'gesture_detector.dart';
+import 'layout_builder.dart';
 import 'ticker_provider.dart';
 
+/// A type for widget builders that take a [Rect] of the current viewport.
+///
+/// See also:
+///
+///   * [InteractiveViewer.builder], whose builder is of this type.
+///   * [WidgetBuilder], which is similar, but takes no Rect.
+typedef TransformedWidgetBuilder = Widget Function(BuildContext context, Rect viewport);
```

Comment:
The name's a little strange. What about this is "transformed"?

Reply:
The return value of this is transformed using a Transform widget by the InteractiveViewer.  Maybe it could be InteractiveViewerWidgetBuilder if that's better?

Reply:
Giving this a more specific name sounds good.